public/javascripts/songClicked.js

```diff
@@ -2,18 +2,17 @@
 var songClicked = (function() {
     var songPlaylistName = m.prop()
     var translationPlaylistName = m.prop()
-    console.log(songPlaylistName())
     var addButtonDOM = m.prop()
     m.mount(document.getElementById('songPlaylistModal'), m(playlistModal.playlistModalComponent, {
         playlistName: songPlaylistName,
         addButtonDOM: addButtonDOM,
         modalName: 'song'
     }))
-    m.mount(document.getElementById('translationPlaylistModal'), m(playlistModal.playlistModalComponent, {
-        playlistName: translationPlaylistName,
-        addButtonDOM: addButtonDOM,
-        modalName: 'translation'
-    }))
+    // m.mount(document.getElementById('translationPlaylistModal'), m(playlistModal.playlistModalComponent, {
```

Comment:
i plan to just have another branch from this to work on the song page layout. just in case i can't make it working in before 6, we still can do user testing without breaking the page